dev-libs/cyrus_sasl/cyrus_sasl-2.1.27~rc7.recipe

```diff
@@ -0,0 +1,90 @@
+SUMMARY="The Cyrus SASL (Simple Authentication and Security Layer)"
+DESCRIPTION="The Cyrus SASL library is a generic library for easy integration of secure network authentication to any client or server application. It supports authentication via standard plaintext methods as well as CRAM-MD5 and DIGEST-MD5 shared secret methods and KERBEROS_V4 and GSSAPI Kerberos methods. The SASL protocol framework is used by SMTP, IMAP, ACAP, LDAP, and other standard protocols."
+HOMEPAGE="http://cyrusimap.web.cmu.edu/"
+COPYRIGHT="1994-2012 Carnegie Mellon University"
+LICENSE="BSD (4-clause)"
+REVISION="1"
+SOURCE_URI="https://github.com/cyrusimap/cyrus-sasl/archive/cyrus-sasl-2.1.27-rc7.tar.gz"
+CHECKSUM_SHA256="5a0c5475b35aad933c0f5939a360cfef86d06dbca93309cb7b223a38a133dc68"
+SOURCE_DIR="cyrus-sasl-cyrus-sasl-2.1.27-rc7"
+PATCHES="cyrus_sasl-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+# On x86_gcc2 we don't want to install the commands in bin/<arch>/, but in bin/.
+commandSuffix=$secondaryArchSuffix
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=
+	commandBinDir=$prefix/bin
+fi
+
+PROVIDES="
+	cyrus_sasl$secondaryArchSuffix = $portVersion compat >= 2
+	cmd:pluginviewer$commandSuffix
+	cmd:saslauthd$commandSuffix
+	cmd:testsaslauthd$commandSuffix
+	lib:libsasl2$secondaryArchSuffix = 3.0.0 compat >= 3
```

Comment:
`libVersionCompat` should be used.

Reply:
> ~`libVersionCompat` should be used.~

Well, it's not a requirement, so it would be better to write:

_You might wish to define a `libVersionCompat` and use it here._
or
_Shall we use a `libVersionCompat` here?_